yamux/src/config.rs

```diff
@@ -38,6 +38,7 @@ pub struct Config {
 
     /// MaxStreamWindowSize is used to control the maximum
     /// window size that we allow for a stream.
+    /// Must be greater than or equal to 256 * 1024
```

Comment:
It's better to use builder to enforce the restriction.

Reply:
Modify the configuration mode when considering the break version number